app/views/template_inputs/_foreign_input_set_form.html.erb

```diff
@@ -0,0 +1,12 @@
+<div class="fields">
+  <div class="form-group">
+    <%= field_set_tag _("Foreign input set").html_safe +
+                          ' ' +
+                          remove_child_link('x', f, {:rel => 'twipsy', "data-title" => _('remove template input set'), :'data-placement' => 'left', :class => 'fr badge badge-danger'}).html_safe do %>
+      <%= selectable_f f, :target_template_id, JobTemplate.all.map { |t| [ t.name, t.id ] }, {}, :class => 'input_type_selector without_select2' %>
+      <%= checkbox_f f, :include_all %>
+      <%= text_f f, :include %>
```

Comment:
inline help with "comma separated values" would be nice I think, same for exclude